org.uqbar.project.wollok.ui/src/org/uqbar/project/wollok/ui/wizard/WollokDslNewProject.xpt

```diff
@@ -28,5 +28,9 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%5p [%t] (%F:%L) - %m%n
 «ENDFILE»
 
+«FILE "src/WOLLOK.ROOT"-»
```

Comment:
Yes it is the root file. 
Maybe it is a good idea to do the change now. 
Now thinking there is a file called wollok.manifest with the exported .wlk files. I will
unify both.